.gitignore

```diff
@@ -1,4 +1,6 @@
 .nyc_output/
 node_modules/
 yarn-error.log
-.env
\ No newline at end of file
+.env
+
+packages/*/dist
```

Comment:
We probably want to keep this included in the repo since we don't have a build hook on our npm publish setup. The npm publish is still being done manually, we should probably setup a Github Action for it at some point.

Reply:
You can add a `prepublishOnly` hook that will run build for you. But there were 900+ files in `dist` directories -- probably don't want those in git.